src/include/btree.i

```diff
@@ -1345,8 +1345,10 @@ __wt_page_can_evict(WT_SESSION_IMPL *session, WT_REF *ref, bool *inmem_splitp)
      * overflow item, because the split into the parent frees the backing blocks for any
      * no-longer-used overflow keys, which will corrupt the checkpoint's block management.
      */
-    if (!__wt_btree_can_evict_dirty(session) && F_ISSET_ATOMIC(ref->home, WT_PAGE_OVERFLOW_KEYS))
+    if (!__wt_btree_can_evict_dirty(session) && F_ISSET_ATOMIC(ref->home, WT_PAGE_OVERFLOW_KEYS)) {
+        WT_STAT_CONN_INCR(session, cache_eviction_fail_overflow_keys);
```

Comment:
This is a rare case and I doubt can happen in MongoDB, since overflow keys are not used there. I think it is okay to have stats around it. But that stat's description needs to be more elaborate around this specific use case.

Reply:
Improved the description of the stat with more details to understand.